recipe/meta.yaml

```
@@ -9,23 +8,24 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: b9662327aa53294c1351b0a9c369093ccec1d21026f050c3bd9b3e5cccf81a86
 
 build:
-  number: 1004
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  number: 0
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - behave = behave.__main__:main
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - setuptools <58   # because of https://github.com/behave/behave/issues/955
                        # which will be released with 1.2.7
     - pip
+    - wheel
   run:
-    - python >=3.6
+    - python
     - parse >=1.6.3
```

```suggestion
    - parse >=1.8.2
```
https://github.com/behave/behave/blob/v1.2.6/setup.py#L71